bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -926,10 +928,23 @@ private function addImplementationNodeFromArrayDefinition(ArrayNodeDefinition $n
                             ->arrayPrototype()
                             ->children()
                                 ->scalarNode('class')->end()
-                                ->arrayNode('matching')
-                                    ->prototype('scalar')->end()
+                                ->booleanNode('translatable')
+                                    ->defaultTrue()
                                 ->end()
-                            ->end()
+                                ->scalarNode('valid_table')->defaultNull()->end()
+                                ->booleanNode('direct_route')
+                                    ->defaultTrue()
```

Comment:
IMHO it should be `defaultFalse`, since prior this PR, it would have required them to be defined in `direct_route_document_types`